.github/workflows/tests.yaml

```
@@ -6,3 +6,10 @@ jobs:
     with:
       context: tests/docker
       name: test
+
+  trivy:
+    needs: bocker-build
+    name: Trivy Vulnerability Scan for Docker images
+    uses: scality/workflows/.github/workflows/trivy.yaml@main
+    with:
+      image-ref: '${{ inputs.registry }}/${{ inputs.namespace }}/${{ inputs.name }}:${{ inputs.tag }}'
```

I'm not sure those context work. Try something like:
```suggestion
      image-ref: 'ghcr.io/${{ github.repository }}/test:${{ github.sha }}'
```